membership-attribute-service/app/models/Attributes.scala

```diff
@@ -131,6 +132,7 @@ object Attributes {
       (__ \ "digitalSubscriptionExpiryDate").writeNullable[LocalDate] and
       (__ \ "paperSubscriptionExpiryDate").writeNullable[LocalDate] and
       (__ \ "guardianWeeklyExpiryDate").writeNullable[LocalDate] and
+      (__ \ "mobileSubscriptionExpiryDate").writeNullable[LocalDate] and
```

Comment:
should this be liveNewsAppSubscriptionExpiryDate - I'm thinking what would we add if we wanted to do the same for Daily Edition in-app purchases?